src/server/routes/garmin.ts

```diff
@@ -1,10 +1,55 @@
 import express from 'express';
+var bytes = require('bytes');
 
+const app = express();
 const garminRouter = express.Router();
+const cors = require('cors');
+const port = 443;                           // listen on this port to data from garmin 
 
-// example
-garminRouter.get('/activities', (req, res) => {
-    return res.status(200).send('result!');
+app.use(express.json({limit: "1mb"}));      // Decode JSON data, unsuccesfully set max size to 1mb
+app.use(express.urlencoded({limit: "1mb", extended: true, parameterLimit:50000})); // Decode Form URL Encoded data, maybe not necessary
+app.use("/", garminRouter);
+app.use(cors());
```

Comment:
good job on handing express advanced properties. just use them in router instead of in app